l10n_cn_hr_payroll/models/l10n_cn_hr_payroll.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico corp (www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import fields, models, api, _
+import openerp.addons.decimal_precision as dp
+from openerp.exceptions import ValidationError
+
+
+class HrContractCn(models.Model):
+    _inherit = 'hr.contract'
+
+    social_insurance_amount = fields.Float(
+        'Social Insurance Base',
+        digits_compute=dp.get_precision('Payroll')
+    )
+    pit_base_amount = fields.Float(
+        'PIT Base',
+        digits_compute=dp.get_precision('Payroll'),
+        default=0
+    )
+    pit_exemption_amount = fields.Float(
+        'Individual Income Tax Threshold',
+        required=True,
+        digits_compute=dp.get_precision('Payroll'),
+        default=3500.00
+    )
+
+    @api.one
+    @api.constrains('pit_base_amount')
+    def _pit_base_amount_check(self):
+        if not self.pit_base_amount > 0:
```

Comment:
not sure about this constraint. I think it should >= not only >.
Besides if you have a check you should have as well the following ones: <= wage
Last but not least SI base should be checked as well.